### DISCUSSION TITLE
添加新的数据库不生效

### DISCUSSION THREAD

**Discussion by phoenixg** [hhyo/Archery#1871] [General]
请问是存在缓存吗？重启了服务仍然不生效

找到 https://github.com/hhyo/Archery/issues/1379 
可是我这边已经过去几个小时了，仍然没生效

在管理后台的“实例数据库列表”里，明明删除界面显示的数据库了，可是前台界面却总看到那个数据库。

尝试注释掉源码里的 archery/sql/instance.py 的 `# @cache_page(60 * 5, key_prefix="insRes")`，重启服务，也不起作用

**Comment by nick2wang** [comment]
“实例数据库列表”只是作为对真实数据库的补充信息，记录负责人和备注用，前台显示的是实例真实存在的数据库，要删除真实数据库请直接在mysql客户端操作

**Reply by phoenixg** [reply]
知道原因了， 看源码发现是保留字符，已解决，谢谢

**Reply by phoenixg** [reply]
顺便问一下，有没有办法把显示数据库的列表细化到指定数据库，而不是实例有的都列出来

**Reply by nick2wang** [reply]
> 顺便问一下，有没有办法把显示数据库的列表细化到指定数据库，而不是实例有的都列出来

你们有特殊需求的话可以二次开发，比如给实例数据库列表加个是否显示的字段
